whipper/common/program.py

```diff
@@ -228,7 +228,7 @@ def getPath(self, outdir, template, mbdiscid, metadata, track_number=None):
             if metadata.releaseType:
                 v['R'] = metadata.releaseType
                 v['r'] = metadata.releaseType.lower()
-            if track_number > 0:
+            if track_number is not None and track_number > 0:
```

Comment:
Is there a reason for not just checking `if track_number and …`? Or do we care that it’s specifically not `False` or an empty string?

Reply:
I'd say, even looking at the old code, that it's not necessary to explicitly check for `not None`.

Reply:
As per your reply to https://github.com/whipper-team/whipper/pull/411#discussion_r342026959 I can see why it might be needed to check whether it’s `None` to prevent a `None` type comparison, so I’m fine with that addition—it just seems like there might be a more elegant way to write it (ie., `if track_number and …`). But maybe this was tried and there was some pitfall there which wasn’t noted in comments. :)

Reply:
```python
track_number = 0
if track_number:  # evaluates to False
```
The only sensible values for `track_number`, as currently implemented, should be `None` and `int`.
In this case `if track_number and track_number > 0` would be fine but the comparison against None is stricter and more explicit.